Taskfile.yaml

```diff
@@ -27,6 +27,11 @@ tasks:
     cmds:
       - docker compose logs --follow
 
+  env-mongosh:
+    desc: "Start mongosh container"
+    cmds:
+      - docker compose up mongosh -d
+
```

Comment:
Okay, so the solution to all of local instance issues is just a new task (which is now in deps of `task dance`). Thanks @rumyantseva for reminding that it's possible to run a single container from docker-compose :slightly_smiling_face:

Reply:
Yeah, the next step would be to not run it at all (as @AlekSi mentioned on Slack).

Reply:
done